lib/api/funnel.js

```diff
@@ -23,6 +23,7 @@
 
 const Bluebird = require('bluebird');
 const Deque = require('denque');
+const Cookie = require('cookie');
```

Comment:
Missing dependency in `package.json`: this only works because this dependency is imported by `winston-elasticsearch`